.github/workflows/ci.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+name: CI
+
+on:
+  pull_request:
+    branches: ["*"]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install Dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y $(find . -name apt-requirements.txt | xargs sed -e "s/#.*//g")
```

Comment:
where is the `apt-requirements.txt` sourced from? I can't see it in the repository

Reply:
It's in `third_party/chromium/apt-requirements.txt` but contains only `gawk` I copied this from the other workflow but might be better to inline `apt install gawk` here.